notifications_utils/field.py

```diff
@@ -204,9 +209,27 @@ def placeholder_names(self):
 
     @property
     def replaced(self):
-        return re.sub(
-            self.placeholder_pattern, self.replace_match, self.sanitizer(self.content)
-        )
+        result = ''
+        block_quote_replace_match = partial(self.replace_match, is_inside_block_quote=True)
+
+        # Process line by line.  This is necessary correctly to handle block quotes.
```

Comment:
grammar. suggest `This is necessary to correctly to handle block quotes.`

Reply:
https://www.grammarly.com/blog/grammar/split-infinitives/

Reply:
I made changes to the comment that included removing "correctly."